src/profile/route.js

```diff
@@ -0,0 +1,71 @@
+import nconf from "../wrio_nconf.js"
+import saveWrioIDForSession from './profiles.js'
+import WrioUsers from '../dbmodels/wriouser.js'
+import db from '../db'
+import {Router} from 'express'
+import {dumpError} from '../utils.js'
+
+const router = Router();
+
+var DOMAIN = nconf.get("db:workdomain");
+
+
+function returndays(j, days, id) {
+    j['url'] = "http://wr.io/" + id + '/';
+    j['cover'] = j['url'] + 'cover.htm';
+    j['days'] = 30 - days;
+    return j;
+}
+
+function returnPersistentProfile(j, id, name) {
+    j['temporary'] = false;
+    j['id'] = id;
+    j['url'] = "http://wr.io/" + id + '/';
```

Comment:
@AlexeyAnshakov i'm afraid that's would break current system operation, better to fix when we will transit to https